migration_to_poc/readme

```diff
@@ -19,6 +19,19 @@
 #############
```

Comment:
@amosh101 please use Markdown syntax here for the readme: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

It is a better way to format the content and github automatically interprets it

Reply:
Moving this comment to a new tech debt refactor ticket